x/crosschain/keeper/abci_test.go

```diff
@@ -360,7 +360,7 @@ func TestCheckAndUpdateCctxGasPrice(t *testing.T) {
 				})
 
 				// ensure median gas price is set
-				medianGasPrice, isFound := k.GetMedianGasPriceInUint(ctx, chainID)
+				medianGasPrice, _, isFound := k.GetMedianGasPriceInUint(ctx, chainID)
```

Comment:
Can we test with some values here?

Reply:
https://github.com/zeta-chain/node/pull/2483/commits/7c16387dd93d3863b09849169d6de3ead7c35b39